setup.py

```diff
@@ -49,6 +49,11 @@
 # add numpy headers
 include_dirs = [np.get_include()]
 
+# check if debug build
+debug_build = "TABMAT_DEBUG" in os.environ
```

Comment:
I would prefer checking for a value for this environment variable. I think the usual value would be 1 for true and 0 for false.

```suggestion
debug_build = os.getenv("TABMAT_DEBUG", False)
# then check that the value makes sense (if the value is "false", this would evaluate to true).
```